style.css

```diff
@@ -599,3 +610,42 @@
     display: none;
   }
 }
+
+@keyframes display {
+  0% {
+    top: -100vh;
+  }
+
+  100% {
+    top: 0;
+  }
+}
+
+@media only screen and (max-width: 375px) {
+  body {
+    width: 100%;
+  }
+
+  .t-heading {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .t-heading h1 {
+    font-family: 'Poppins', sans-serif;
+    color: #172b4d;
+    font-weight: bolder;
+    font-size: 32px;
+    padding: 0 5px;
+    z-index: 0;
+  }
+
+  .socials {
+    display: flex;
+  }
+
+  .socials a {
+    padding: 0 7px;
+    margin: 2px;
+  }
+}
```

Comment:
Your popup is responsive 👏🏽  but the `blur` background is not taking the whole screen which is kind of messing up with the design 🤧 Look to adjust that, this is what you have so far 👇🏽 
![Screenshot 2023-03-16 at 08 44 11](https://user-images.githubusercontent.com/45739963/225620795-bcfb222d-5458-4234-9b47-d992db437906.png)


The projects section added dynamically is also broken for now on mobile version, mainly because of the image size it seems 👇🏽 
![Screenshot 2023-03-16 at 08 43 13](https://user-images.githubusercontent.com/45739963/225620558-6209fa63-522e-4c31-b14a-9cf166bfdc46.png)